src/finder/mod.rs

```diff
@@ -119,7 +119,7 @@ impl FinderChoice {
         ]);
 
         if !opts.show_all_columns {
-            command.args(["--with-nth", "1,2,3"]);
+            command.args(["--read0", "--print0", "--with-nth", "1"]);
```

Comment:
It is better to rename `show_all_columns` or introduce another option for `--read0` and `--print0` because `--read0` and `--print0` do not control the number of columns to display.